### PR TITLE
Populated Test Matrix topic

### PR DESCRIPTION
**Pull request by PeterTurcan**

### CODE REVIEW

**Comment by PeterTurcan** [comment]
@alandefreitas can this PR be merged?

**Comment by alandefreitas** [comment]
I think I never reviewed it. Let me have a look.

File: contributor-guide/modules/ROOT/pages/testing/boost-test-matrix.adoc [hunk 1]
```diff
@@ -1,4 +1,74 @@
...
+
+=== Developer Report
+
+If you select the **Report info** entry, top left on the left-hand column, you can see some metrics for the overall test process:
+
+image::test-matrix-developer-report.png[]
+
+== Boost Build
+
+The Test Matrix uses https://www.bfgroup.xyz/b2/[B2] as its build system. Understanding how to write and modify Jamfiles (Boost.Build scripts) can be helpful for integrating your tests into the matrix.
```
Comment (alandefreitas) [inline contributor-guide/modules/ROOT/pages/testing/boost-test-matrix.adoc:61]: Maybe there could be a B2 primer focused on boost contributors. Most people know CMake but they don't know B2 and just end up copying and pasting.

**Review by alandefreitas — APPROVED** [review]

**Comment by alandefreitas** [comment]
Great content.

**Pull request merged**